### PR TITLE
More clang-tidy clean-ups

### PR DESCRIPTION
**Pull request by yperbasis**

### CODE REVIEW

File: node/silkworm/etl/file_provider.cpp [hunk 1]
```diff
@@ -104,8 +104,4 @@ void FileProvider::reset() {
     }
 }
 
-std::string FileProvider::get_file_name(void) const { return file_name_; }
-
-size_t FileProvider::get_file_size(void) const { return file_size_; }
-
```
Comment (AndreaLanfranchi) [inline node/silkworm/etl/file_provider.cpp:110]: Please don't remove these. They're propedeutic to integration-2 where I implemented the total size and logging of collector

File: node/silkworm/etl/file_provider.hpp [hunk 2]
```diff
@@ -33,14 +33,12 @@ namespace silkworm::etl {
...
     FileProvider(std::string file_name, size_t id);
-    ~FileProvider(void);
+    ~FileProvider();
+
     void flush(Buffer& buffer);                            // Write buffer's contents to disk
     std::optional<std::pair<Entry, size_t>> read_entry();  // Read next data element from file starting from position 0
     void reset();                                          // Remove the file when eof is met
 
-    std::string get_file_name(void) const;
-    size_t get_file_size(void) const;
```
Comment (AndreaLanfranchi) [inline node/silkworm/etl/file_provider.hpp:42]: See comment above

**Review by AndreaLanfranchi — CHANGES_REQUESTED** [review]

File: node/silkworm/etl/file_provider.cpp (hunk 1 above)
Comment (yperbasis) [reply node/silkworm/etl/file_provider.cpp]: Sure, restored.

File: node/silkworm/etl/file_provider.hpp (hunk 2 above)
Comment (yperbasis) [reply node/silkworm/etl/file_provider.hpp:42]: 👍

**Review by mriccobene — APPROVED** [review]

**Review by AndreaLanfranchi — APPROVED** [review]

**Pull request merged**